infrateam_first_responder.md

```diff
@@ -116,9 +116,9 @@ At the very least, the first responder should be watching incoming alerts from H
 Other places to monitor:
 * Nagios alerts (https://sul-nagios-prod.stanford.edu/nagios/  The Services tab on the left column will give overview of checks.  The Problems->Services tab will give you what’s alerting.)
 * feedback email lists, e.g. argo-feedback, sinopia-feedback, etc
-* the `#dlss-infrastructure` channel
+* the `#dlss-infrastructure`, channels
```

Comment:
it seems "channels" plural applies to your new line 121 but not to this line ...

more channels (ugh):

dlss-etds-dev
dlss-preservation-dev

Reply:
Updated and added this channels. Reformatted to a markdown list.